src/client/src/metrics.rs

```diff
@@ -22,4 +22,5 @@ pub const METRIC_GRPC_LOGICAL_PLAN: &str = "grpc.logical_plan";
 pub const METRIC_GRPC_ALTER: &str = "grpc.alter";
 pub const METRIC_GRPC_DROP_TABLE: &str = "grpc.drop_table";
 pub const METRIC_GRPC_FLUSH_TABLE: &str = "grpc.flush_table";
+pub const METRIC_GRPC_COMPACT_TABLE: &str = "grpc.compact_table";
```

Comment:
off-topic: So many metrics 😢 Maybe we should distinguish them by label instead of metric names.

Reply:
> off-topic: So many metrics 😢 Maybe we should distinguish them by label instead of metric names.

Maybe in a separate PR.

Reply:
Consider good first issue